### PR TITLE
docs: cn: jp: Guard kernel headers includes in example

### PR DESCRIPTION
**Pull request by danobi**
Same change as #2919 .

##### Checklist

- [ ] Language changes are updated in `man/adoc/bpftrace.adoc` and if needed in `docs/reference_guide.md`
- [x] User-visible and non-trivial changes updated in `CHANGELOG.md`
- [ ] The new behaviour is covered by tests

### CODE REVIEW

**Comment by danobi** [comment]
cc @Rtoax @mmisono @Hagi013 (who last touched these files, even if it was a while ago)

File: docs/tutorial_one_liners_japanese.md [hunk 1]
```diff
@@ -318,7 +320,7 @@ open path: retrans_time_ms
 - kprobe: 以前説明したように，これはカーネルの動的トレーシングをおこなうプローブタイプで，カーネル関数の開始をトレースします（関数からの戻りのトレースには kretprobe を利用します）．
 - `arg0` はビルトイン変数で，プローブの最初の引数を意味します．これはプローブタイプごとに意味が異なり，`kprobe` の場合は関数の最初の引数を意味します．他の引数には arg1, ..., argN でアクセスできます．
 - `((struct path *)arg0)->dentry->d_name.name`: `arg0` を `struct path *` にキャストしてから dentry や後続のメンバ変数を参照します．
-- #include: path と dentry の構造体定義のために必要なファイルをインクルードします．
+- #include: BTF (BPF Type Format) を持たないシステムでの path および dentry 構造定義に必要なファイルをインクルードします。
```
Comment (mmisono) [inline docs/tutorial_one_liners_japanese.md:323]:
```suggestion
- #include: BTF (BPF Type Format) を持たないシステムで path および dentry 構造定義に必要なファイルをインクルードします。
```

Comment (danobi) [reply docs/tutorial_one_liners_japanese.md:323]: thanks for taking a look!

**Comment by danobi** [comment]
I'm gonna go ahead and merge. Feel free to still add comments, though

**Pull request merged**